src/components/timeline/TimelineCanvas.js

```diff
@@ -178,33 +187,3 @@ export default class TimelineCanvas<HoveredItem> extends PureComponent<
     );
   }
 }
-
-/**
- * Check for shallow equality for objects, and strict equality for everything else.
- */
-function hoveredItemsAreEqual(a: any, b: any) {
```

Comment:
I'm wondering if this is still needed. I'm getting re-draws every mouse move when I'm hovering over an item and it hasn't changed.